proto/buf/alpha/registry/v1alpha1/reference.proto

```diff
@@ -64,6 +65,11 @@ service ReferenceService {
   rpc ListGitCommitMetadataForReference(ListGitCommitMetadataForReferenceRequest) returns (ListGitCommitMetadataForReferenceResponse) {
     option idempotency_level = NO_SIDE_EFFECTS;
   }
+  // ListGitCommitsNoMetadataForReference takes a string reference and returns all the git commit
+  // labels without metadata associated with the resolved reference commit.
+  rpc ListGitCommitsNoMetadataForReference(ListGitCommitsNoMetadataForReferenceRequest) returns (ListGitCommitsNoMetadataForReferenceResponse) {
```

Comment:
I assume this RPC would be invoked when `ListGitCommitMetadataForReference` returns empty for a given reference?

Reply:
Yeah, so in the world where a user has `buf push`'d a BSR commit, that BSR commit reference will not have any git metadata, but may still have meaningful `GIT_COMMIT_SHA` labels that need to be displayed, so this RPC would be invoked for that circumstance.